apps/bfd-model-idr/maps/Coverage-Base.map

```diff
@@ -167,6 +167,17 @@ group createPartDCoverage(source src: CoverageBase, target tgt: BFDCoverage){
         src -> tgtExtension.value = pbpNum "Add value";
     } "Add PBP number";
 
+    //This can be obtained from the contract + PBP by joining V2_MDCR_CNTRCT_PBP_SGMT with V2_MDCR_CNTRCT_PBP_NUM on CNTRCT_PBP_SK. 
```

Comment:
I don't understand this comment. The dictionary support files indicates that `CNTRCT_PBP_SGMT_NUM` has source view = `V2_MDCR_CNTRCT_PBP_SGMT`.  How should `V2_MDCR_CNTRCT_PBP_NUM` be used?